src/main/java/com/arpnetworking/http/Routes.java

```diff
@@ -135,13 +149,25 @@ public Routes(
                                                         "{\"status\":\""
                                                                 + (isHealthy ? HEALTHY_STATE : UNHEALTHY_STATE)
                                                                 + "\"}")));
-            } else if (_statusPath.equals(request.getUri().path())) {
+            } else if (_statusPath.equals(path)) {
                 return CompletableFuture.completedFuture(
                         HttpResponse.create()
                                 .withStatus(StatusCodes.OK)
                                 .withEntity(JSON_CONTENT_TYPE, ByteString.fromString(STATUS_JSON)));
             }
+        } else if ((HttpMethods.POST.equals(request.method()) || HttpMethods.PUT.equals(request.method()))
```

Comment:
Are you sure you want to accept PUT? I believe this typically implies idempotence which we do not support and likely will not for some time.